easybuild/easyconfigs/l/libdrm/libdrm-2.4.27-intel-2014b.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'libdrm'
+version = '2.4.27'
+
+homepage = 'http://dri.freedesktop.org'
+description = """Direct Rendering Manager runtime library."""
+
+source_urls  = ['http://dri.freedesktop.org/libdrm/']
+sources = [SOURCELOWER_TAR_GZ]
+
+toolchain = {'name': 'intel', 'version': '2014b'}
+
+dependencies = [
+    ('libpthread-stubs', '0.3'),
+    ('libpciaccess', '0.13.1'),
+]
+
+configopts = '--enable-nouveau-experimental-api'
```

Comment:
disable it. I cannot image why we would need it and you disable all gallium drivers in Mesa anyway